src/components/Stakeholders/Table.js

```diff
@@ -0,0 +1,119 @@
+import React, { useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  CircularProgress,
+  Paper,
+  TableContainer,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  TableSortLabel,
+  TablePagination,
+} from '@material-ui/core';
+import StakeholderDetail from './StakeholderDetail';
+import { StakeholdersContext } from '../../context/StakeholdersContext';
+
+const useStyles = makeStyles({
+  placeholder: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    textAlign: 'center',
+    height: '60vh',
+    width: '100vw',
+  },
+});
+
+function StakeholderTable() {
+  const classes = useStyles();
+  const {
+    stakeholders,
+    count,
+    columns,
+    isLoading,
+    page,
+    orderBy,
+    order,
+    rowsPerPage,
+    setPage,
+    setRowsPerPage,
+    sort,
+  } = useContext(StakeholdersContext);
+
+  const handleRowsPerPageChange = (e) => {
+    setRowsPerPage(e.target.value);
+  };
+
+  const handlePageChange = (e, page) => {
+    setPage(page);
+  };
+
+  const handleSort = (col, order) => {
+    console.log('sort col', { col, order });
+    sort({ col, order });
+  };
+
+  return (
+    <>
+      {isLoading ? (
+        <div className={classes.placeholder}>
+          <CircularProgress id="loading" />
+        </div>
+      ) : (
+        <TableContainer component={Paper}>
+          <Table aria-label="collapsible table">
+            <TableHead>
+              <TableRow>
+                {columns.map((col) => (
+                  <TableCell
+                    key={col.value}
+                    onClick={() => handleSort(col.value, !order)}
+                  >
+                    <TableSortLabel
+                      active={col.value === !!orderBy}
```

Comment:
As a user, I'd like to understand the data that I'm currently looking at in the table. So if a sort is applied, I'd prefer for the arrow indicator to remain. Changing L75 to `active={col.value === orderBy}` persists the arrow label rather than having it unmount immediately.  This may be useful, especially since sort is retained when changing pages.

Reply:
Done!